app/controllers/administrateurs/procedure_labels_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Administrateurs
+  class ProcedureLabelsController < AdministrateurController
+    before_action :retrieve_procedure
+    before_action :set_colors_collection, only: [:edit, :new, :create, :update]
+
+    def index
+      @labels = @procedure.procedure_labels
```

Comment:
alors c'est surement trop tard pour dire ca, mais j'ai l'impression que ca veut dire que `ProcedureLabel` devrait être nommé `Label`, en plus, ca allégerait la lecture un truc de ouf !

Enfin, moi je dis ça ...

Et si tu trouves l'idée bonne, je veux bien m'y coller a faire un gros search and replace bien délicat.